definitions/npm/react-navigation_v2.x.x/flow_v0.60.x-/react-navigation_v2.x.x.js

```diff
@@ -533,7 +533,16 @@ declare module 'react-navigation' {
       eventName: string,
       callback: NavigationEventCallback
     ) => NavigationEventSubscription,
-    getParam: (paramName: string, fallback?: any) => any,
```

Comment:
Hi, 
maybe add a test?  
Also because, if I'm not mistaken, calling function/methods with generics is not available before Flow 0.72.

Reply:
@pascalduez the function type parameter is just not exposed outside prior that.